images/kube-vip/README.md

```diff
@@ -0,0 +1,124 @@
+<!--monopod:start-->
+# kube-vip
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/kube-vip` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/kube-vip/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Kubernetes Control Plane Virtual IP and Load-Balancer
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/kube-vip:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+
+## Usage
+
+You can test the kube-vip in several ways. Here is an example of how to deploy the kube-vip controller into KinD (Kubernetes in Docker) cluster:
+
+Find Address Pool for kube-vip:
+
+```bash
+docker network inspect kind -f '{{ range $i, $a := .IPAM.Config }}{{ println .Subnet }}{{ end }}'
+```
+
+This will return a CIDR range such as `172.18.0.0/16` which you can use as the `range-global` for the kube-vip ConfigMap.
```

Comment:
Is there another step to use this info below - i don't see anywhere we're patching or creating a ConfigMap with it?